client/src/components/AddTask.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Grid, TextField } from "@mui/material";
+import { useState } from "react";
+import NewTaskMutation from "./schema/mutations/NewTaskMutation";
+
+export default function AddTask(props) {
```

Comment:
we try to type our functional components:
```suggestion
interface Props {
...
}

const AddTask: React.FC<Props> = (props) => { ... }

export default AddTask;
```